src/args.rs

```diff
@@ -0,0 +1,30 @@
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+pub struct PokerArgs {
+    #[command(subcommand)]
+    pub command: Option<CommandsEnum>,
+}
+
+#[derive(Debug, Subcommand)]
+pub enum CommandsEnum {
+    /// Draw random hand, display, and score
+    DrawHand{hands_size: usize,},
+
+    /// Compute statistics on a 5/7 card hand with N samples
+    Statistics(StatisticsSampleParameters),
+
+    /// To print sorted deck
+    SortedDeck,
+}
+
+#[derive(Debug, Args)]
+pub struct StatisticsSampleParameters {
+    /// Number of cards
+    pub hands_number: usize,
+
+    /// Number of samples
+    pub sample_number: u32,
```

Comment:
nit:
```suggestion
    pub number_of_samples: u32,
```

Can we drop the comment because the code is self-documenting? Or does the CLI stuff need it?